prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    fullName = 'Cristi Neames'
+    console.log(fullName)
+    numberOfPets = 4
+    console.log(numberOfPets)
+    likesVegetables = true
+    console.log(likesVegetables)
+    nothing = null
```

Comment:
Be sure to declare your variables using the `var` keyword. The code works without it, but it also does some unexpected things! 👀 

```js
var fullName = 'Cristi Neames';
// etc
```